scripts/dev.sh

```diff
@@ -12,7 +12,7 @@ echo "Starting @ $(date)" > $PATH_ROOT/logs/dev.txt
 
 # run flask
 cd src/api/
-flask run --reload >> $PATH_ROOT/logs/api.txt 2>&1 & # run in bg and port logs
+DB_USER=marchdan DB_PASS=" " flask run --reload >> $PATH_ROOT/logs/api.txt 2>&1 & # run in bg and port logs
```

Comment:
Let's not add this to master lol. we need .env files

Reply:
XD I will fix that